src/main/java/org/example/service/MenuService.java

```diff
@@ -0,0 +1,216 @@
+package org.example.service;
+
+import org.example.enums.Card;
+import org.example.enums.MenuMessages;
+import org.example.model.User;
+import org.example.repository.AccountRepositoryImpl;
+import org.example.repository.GameRepositoryImpl;
+import org.example.model.Game;
+import org.example.repository.UserRepositoryImpl;
+
+import java.sql.Connection;
+import java.util.InputMismatchException;
+import java.util.List;
+import java.util.Scanner;
+
+public class MenuService {
+
+    private Scanner scanner;
+    private Connection connection;
+    private GameService gameService;
+    private UserService userService;
+    private AccountService accountService;
+    private User user = null;
+
+
+    public MenuService(Scanner scanner, Connection connection) {
+        this.scanner = scanner;
+        this.connection = connection;
+        this.gameService = new GameService(new GameRepositoryImpl(this.connection));
+        this.userService = new UserService(new UserRepositoryImpl(this.connection));
+        this.accountService = new AccountService(new AccountRepositoryImpl(this.connection));
+    }
+
+    private void welcome() {
+        System.out.println(MenuMessages.WELLCOME.getMessage());
+        String answer = scanner.next();
+        if (answer.equals("1"))
```

Comment:
The best practice is to call equals to not null value. I mean "1".equals(answer). 
Of course, in the current case scanner.next() never returns null.
 "1".equals(answer) will be good if in the future it is necessary to change how the value of the "answer" will be obtained.

Reply:
Thanks, changed it